docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -105,12 +105,12 @@ GET /_cluster/allocation/explain
 
```

Comment:
This should address the needed changes @DaveCTurner mentioned. This snippet (or something like it) is required to pass the CI tests.

```suggestion
//////
[source,console]
--------------------------------------------------	
DELETE /myindex	
--------------------------------------------------	
//////

```